README.md

```diff
@@ -30,12 +30,23 @@ Requirements:
 
 - go 1.6 or higher
 - A [working golang](https://golang.org/doc/code.html) environment
+- A [properly configured](https://golang.org/doc/code.html#Organization) Go workspace
 
-From the project root directory, run:
+In your terminal, run:
 ```
+cd $GOPATH
```

Comment:
Please offer a default  $GOPATH instruction?

I.e... if this is your first time with GO, then type in:

```
export GOPATH=~/go
mkdir -p $GOPATH
```